extra-i18n/fcitx5-rime/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=fcitx5-rime
+PKGDES="Fcitx5 wrapper for librime"
+PKGDEP="fcitx5 boost librime opencc brise"
```

Comment:
Explain why `opencc` is required.

Reply:
> Explain why `opencc` is required.

`brise` need `opencc`?....ok, actually can remove it...

Reply:
and maybe need `curl` user for download cloud pinyin etc...

Reply:
>brise need opencc?....ok, actually can remove it...

 No, `brise` does not require `opencc`. It is `librime` whose dependencies include `opencc`.